Singularity/Singularity/Screen/Button.cs

```diff
@@ -152,24 +196,46 @@ protected virtual void OnButtonClicked()
         /// <param name="spriteBatch"></param>
         public void Draw(SpriteBatch spriteBatch)
         {
-            if (Active)
+            if (ActiveWindow)
             {
                 // draw for button that uses a Texture2D
                 if (mIsText == false)
                 {
-                    spriteBatch.Draw(mButtonTexture,
-                        Position,
-                        null,
-                        mColor * Opacity,
-                        0f,
-                        new Vector2(0, 0),
-                        mScale,
-                        SpriteEffects.None,
-                        0f);
-                    if (mWithBorder)
+                    // draw for buttons which need to crop the texture
```

Comment:
which buttons need to crop texture? and if so, why do they need to crop textures?

Reply:
The button gets a texture, but since the platforms texture got all the platforms in it I must tell the button which part to draw or else the entire texture is drawn

Reply:
Cropping is probably the wrong word. It's the typical 'chose a part of the texture using a rectangle' way..

Reply:
since the textures have changed, make sure this crops them properly